client/js/place.js

```diff
@@ -456,6 +457,34 @@ var place = {
         this.didSetHash = true;
     },
 
+    contextMenu: function(event) {
+        event.preventDefault();
+        let zoom = place._getZoomMultiplier();
+        let x = Math.round((event.pageX - $(place.cameraController).offset().left) / zoom)
+        let y = Math.round((event.pageY - $(place.cameraController).offset().top) / zoom)
```

Comment:
`this` doesn't work. `this` ends up being the canvas' element. I tried it to no avail